scripts/tests/provisioning/configure_vm.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+#
+# This script helps with configuring a QEMU VM
+# It:
+#  * boots an already-provisioned NILRT QEMU virtual machine.
+#  * configures opkg with a locally-hosted ipk feed.
+#  * runs the provided expect file
+set -euo pipefail
+
+readonly SCRIPT_ROOT=$(dirname "$BASH_SOURCE")
+
+
+## UTILITY FUNCTIONS
+
+error_and_die() {
+	echo "ERROR: " $@ >&2
+	exit 1
+}
+
+
+## ARGUMENT PARSING AND ENVIRONMENT
+
+usage() {
+	local exit_code=${1:-2}
+	test $exit_code -eq 0 || exec 1>&2
+	cat <<EOF
+$(basename $BASH_SOURCE) -h|--help
+  Print this usage information and exit.
+$(basename $BASH_SOURCE) [-i ipk_feed_dir] nilrt_vm_path
+  Start the NILRT VM and run the provisioning tests.
+
+Options:
+-i|--ipk-feed  Path to a local IPK feed, which will be added to the VM's opkg
+               configuration.
+
+Arguments:
+nilrt_vm_path  Path to an already-provisioned NILRT VM directory. (See the
+               buildVM.sh script in the NILRT.git source.)
+EOF
+	exit $exit_code
+}
+
+readonly IPK_FEED_PORT=8990
+readonly EXPECT=/usr/bin/expect
+
+# Check environment requirements
+test -x "$EXPECT" || error_and_die "$EXPECT not found. This script requires the tcl-expect package to instrument test operations."
+
+VERBOSE=${VERBOSE:-}
+ipk_feeds=()
+positionals=()
+
+while test -n "${1:-}"; do case "$1" in
+	-h|--help)
+		usage 0
+		;;
+	-i|--ipk-feed)
+		ipk_feeds+=("$2")
+		shift
+		shift
+		;;
+	-*|--*)
+		echo "Invalid or unrecognized option: $1"
+		usage 2
+		;;
+	*)
+		positionals+=("$1")
+		shift
+		;;
+esac; done
+
+if [ ${#positionals} -lt 1 ]; then
```

Comment:
Should this be '-lt 2' now that the VM path and expect file are positional params?